edk2toolext/rust_environment.py

```diff
@@ -0,0 +1,442 @@
+# @file rust_environment.py
```

Comment:
I feel like this file would make more sense in `edk2toolext/environment/*`. Thoughts? I'm not 100% sold on that thought.

Reply:
I think that makes sense. I moved it.